iib/workers/tasks/utils.py

```diff
@@ -68,7 +68,7 @@ def _add_property_to_index(db_path: str, property: Dict[str, str]) -> None:
     con.close()
 
 
-def add_max_ocp_version_property(resolved_bundles: List[str], temp_dir: str) -> None:
+def add_max_ocp_version_property(resolved_bundles, temp_dir):
```

Comment:
Can you please keep the typing annotation?

Reply:
Nice catch! Thanks